azure-pipelines.yml

```diff
@@ -153,7 +153,8 @@ jobs:
     dependsOn: [git_commit, linting]
     condition: |
       and(
-        succeeded(),
+        # Runs when dependencies succeeded or skipped
+        not(or(failed(), canceled())
```

Comment:
Oops, I changed this line by mistake. Let's wait a bit before accepting this submission. I would like to test the cancellation on the other steps with an empty commit first.

```suggestion
        succeeded(),
```

Reply:
ok I forgot some comas, will push a new commit.